### PR TITLE
Add login screen

### PR DESCRIPTION
**Pull request by hututudor**
Jira: https://etvas.atlassian.net/browse/ETK-34

### CODE REVIEW

File: src/Card/Card.variants.js [hunk 1]
```diff
@@ -3,5 +3,8 @@ export default {
   popup: {
     border: '1px solid',
     borderColor: 'lighterOutline'
+  },
+  tainted: {
```
Comment (alsofronie) [inline src/Card/Card.variants.js:7]: Please use:
```suggestion
  tinted: {
```

File: src/Form/TextField.jsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ export const TextField = props => {
       id={id}
       error={meta.touched && meta.error}
       valid={hasValidation(props) && !meta.error && meta.touched}
+      tainted={props.tainted}
```
Comment (alsofronie) [inline src/Form/TextField.jsx:20]:
```suggestion
      tinted={props.tinted}
```

File: src/Form/TextField.jsx [hunk 3]
```diff
@@ -27,9 +28,11 @@ TextField.propTypes = {
   ...fieldShape,
   type: PropTypes.string,
   placeholder: PropTypes.node,
-  required: PropTypes.bool
+  required: PropTypes.bool,
+  tainted: PropTypes.bool
```
Comment (alsofronie) [inline src/Form/TextField.jsx:32]:
```suggestion
  tinted: PropTypes.bool
```

File: src/Form/TextField.jsx [hunk 4]
```diff
@@ -27,9 +28,11 @@ TextField.propTypes = {
...
 }
 
 TextField.defaultProps = {
-  type: 'text'
+  type: 'text',
+  tainted: false
```
Comment (alsofronie) [inline src/Form/TextField.jsx:37]:
```suggestion
  tinted: false
```

File: stories/LoginScreen.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+  }
+  if (!values.password) {
+    errors.password = 'Required'
+  }
+
+  if (values) return errors
+}
+
+export const Default = () => <LoginScreen />
+export const Tainted = () => <LoginScreen tainted />
```
Comment (alsofronie) [inline stories/LoginScreen.stories.jsx:39]:
```suggestion
export const Tinted = () => <LoginScreen tinted />

```

**Review by alsofronie — CHANGES_REQUESTED** [review]

**Review by alsofronie — CHANGES_REQUESTED** [review]
Please replace `tainted` with `tinted` in all your source code, not only where i've spotted them

**Comment by hututudor** [comment]
Now it should be ok. Checked with grep.

**Review by alsofronie — APPROVED** [review]

**Pull request merged**